assets/src/domain/eventEditor/ui/datetimes/datesList/tableView/TableView.tsx

```diff
@@ -2,25 +2,24 @@ import React from 'react';
 import classNames from 'classnames';
 import { __ } from '@wordpress/i18n';
 
+import { DatesListViewProps } from '../types';
 import datesListTableHeader from './datesListTableHeader';
 import datesListTableRow from './datesListTableRow';
 import { ResponsiveTable } from '@appLayout/espressoTable';
-import { useDatetimeItem } from '@edtrServices/apollo/queries';
-
-import { TableViewProps } from './types';
 
 import './styles.scss';
 
-const noZebraStripe = ['row', 'stripe', 'name', 'actions'];
+// const noZebraStripe = ['row', 'stripe', 'name', 'actions'];
 
 /**
  * EditorDateEntitiesListView
  * Displays event date details in a standard list table like view
  */
-const TableView: React.FC<TableViewProps> = ({ className, displayStartOrEndDate, entities: datetimes, ...props }) => {
-	const formRows = datetimes.map(({ id }) => {
-		const datetime = useDatetimeItem({ id });
-		const columns = datesListTableRow({ datetime, displayStartOrEndDate, ...props });
```

Comment:
I am surprised that a hook is used inside a callback. This mistake has been made many times now. We need to get ESLint working for react-hook-rules

Reply:
oh yuk... ya i totally missed that in my reviews
good catch

i think this was done because the entities were not refreshing after changes were made to them, so we'll have to watch out to make sure that they are not displaying stale copies